examples/stereo_depth_from_host.py

```diff
@@ -12,8 +12,14 @@
 datasetDefault = str((Path(__file__).parent / Path('models/dataset')).resolve().absolute())
 parser = argparse.ArgumentParser()
 parser.add_argument('-dataset', nargs='?', help="Path to recorded frames", default=datasetDefault)
+parser.add_argument('-debug', "--debug", action="store_true", help="Enable debug outputs.")
```

Comment:
Not sure, why this is showing up as change, it's already in develop.
Flatten the commit history of this branch, from 26 commits to 1.
Not sure what approach you took, but commits are duplicated if you check [commit history](https://github.com/luxonis/depthai-python/pull/379/commits).

Reply:
So, as far as I understand, you added this to the `develop` branch 3 days ago, but I made this `gen2_mesh_api` branch ~5 days ago from the `main` branch. Since I rebased this just a few hours ago, all the changes from `develop` branch from 5 days ago to now were merged. I will fix the commit history now.